src/Sulu/Bundle/ContentBundle/Controller/NodeController.php

```diff
@@ -463,9 +463,13 @@ private function cgetContent(Request $request)
         $excludeGhosts = $this->getBooleanRequestParameter($request, 'exclude-ghosts', false, false);
         $excludeShadows = $this->getBooleanRequestParameter($request, 'exclude-shadows', false, false);
         $webspaceNodes = $this->getRequestParameter($request, 'webspace-nodes');
-        $locale = $this->getRequestParameter($request, 'language', true);
+        $locale = $this->getRequestParameter($request, 'locale', false, $this->getRequestParameter($request, 'language', false));
```

Comment:
Haven't you already implemented something similar in that controller? If we move that to a separate function we wouldn't have to repeat ourselves, and we could add a deprecation notice using `trigger_error` if the old parameter is used.